api/src/main/java/com/epam/pipeline/manager/datastorage/DataStorageApiService.java

```diff
@@ -199,9 +200,9 @@ public DataStorageDownloadFileUrl generateDataStorageItemUploadUrl(Long id, Stri
     }
 
     @PreAuthorize(STORAGE_ID_OWNER)
-    public void restoreFileVersion(Long id, String path, String version)
+    public void restoreFileVersion(Long id, String path, String version, RestoreFolderVO restoreFolderVO)
```

Comment:
May be we should rename this method due to the fact that we support folders?

Reply:
added separate method restoreFolderVersion()